### PR TITLE
Migration: Delete Room model in timetable

### PR DESCRIPTION
**Pull request by zipy124**
We migrated to a standard room model awhile ago, it seems I forgot to make the migration for it. Before we had a rooms model in timetable and a rooms model in /rooms.

### CODE REVIEW

File: backend/uclapi/requirements.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ idna==2.9
 jinjasql==0.1.7
 jmespath==0.9.5
 jsonpickle==1.3
-kombu==4.6.3
```
Comment (Feverfew) [inline backend/uclapi/requirements.txt]: Unrelated change

File: backend/uclapi/timetable/migrations/0019_delete_rooms.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.10 on 2020-03-29 16:56
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('timetable', '0018_classificationsa_classificationsb'),
```
[the remark this answers is not in the document]
Comment (Feverfew) [reply backend/uclapi/timetable/migrations/0019_delete_rooms.py]: What on earth is this?

[the remark this answers is not in the document]
Comment (zipy124) [reply backend/uclapi/timetable/migrations/0019_delete_rooms.py]: @Feverfew I've no idea what this is but its pretty hilarious
